package.json

```diff
@@ -31,6 +31,8 @@
         "@testing-library/user-event": "^14.5.2",
         "@types/pg": "^8.11.10",
         "@types/react": "18.3",
+        "@typescript-eslint/eslint-plugin": "^8.13.0",
+        "@typescript-eslint/parser": "^8.13.0",
```

Comment:
curious why we needed to add these, was lint not working correctly for you?

Reply:
I see the commit message now, that makes sense to silence the unknown TS version warning

Reply:
I was getting this error which would not go away with a simple `npm install`:
```WARNING: You are currently running a version of TypeScript which is not officially supported by @typescript-eslint/typescript-estree.
You may find that it works just fine, or you may not.
SUPPORTED TYPESCRIPT VERSIONS: >=4.7.4 <5.6.0
YOUR TYPESCRIPT VERSION: 5.6.2
```

It's 99% a my-side problem which maybe does not need to get into code. I'm totally okay with reverting it because the CI does not have this errors it seems with a fresh install.